docs/content/en/docs/clustermgmt/observability/cluster-connect.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: "Connect cluster to console"
+title: "Connect EKS Anywhere cluster to console"
 linkTitle: "Connect to console"
 weight: 80
 aliases:
     /docs/reference/baremetal/bare-custom-hookos/
 date: 2017-01-05
 description: >
-  Connect a cluster to the EKS console
+  Connect an EKS Anywhere cluster to the EKS console
 ---
 
-The AWS EKS Connector lets you connect your EKS Anywhere cluster to the AWS EKS console, where you can see your the EKS Anywhere cluster, its configuration, workloads, and their status.
-EKS Connector is a software agent that can be deployed on your EKS Anywhere cluster, enabling the cluster to register with the EKS console.
+The AWS EKS Connector lets you connect your EKS Anywhere cluster to the AWS EKS console. The connected console displays the EKS Anywhere cluster, its configuration, workloads, and their status.
+EKS Connector is a software agent that can be deployed on your EKS Anywhere cluster that enables the cluster to register with the EKS console.
```

Comment:
"EKS Connector is a software agent that runs on your EKS Anywhere cluster and registers the cluster with the EKS console"